assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -41,17 +42,37 @@ function Panel({
   children,
   resizeable = false,
   canCollapse = true,
+  collapsedByDefault = true,
   initialHeight = null,
   ariaLabel = null,
   ariaHidden = false,
+  isPersisted = true,
```

Comment:
I'm not sure I like the name of this prop. It sounds like it talks about whether it _has been_ persisted, not about whether it _can be_ persisted.

`isPersistable` or something like that might be more transparent down the line (it's actually a **real** word!).